### ISSUE TITLE
Displayed model schema should match the EDM Model

### ISSUE THREAD

**Issue by bigred8982** [rbeauchamp/Swashbuckle.OData#59]
When using automatic property name conversion via the EnableLowerCamelCase() method on the ODataConventionModelBuilder, the models displayed in the Swagger UI do not match the models in the payload.

For example, the C# model will have a property called 'Id' The service is returning the property as 'id' in the payload. The *Model* and *Model Schema* sections are displaying the C# property name in Pascal case.

I have prepared an example to illustrate the issue. Please refer to *Swashbuckle.OData.Sample* project in the branch below:

[https://github.com/bigred8982/Swashbuckle.OData/tree/property_casing_sample]()

**Comment by rbeauchamp** [comment]
Thanks Harrison, I'll checkout your sample.

**Comment by rbeauchamp-lendingtree** [comment]
@bigred8982 Fix is available in v2.12.1. Give it a shot and let me know if you find any issues. Thanks!

**Comment by bigred8982** [comment]
The casing has definitely been taken care of.

I discovered a related issue. The model may still not match what is in the EDM is you customize the EDM model with *ignored* properties. I haven't had the chance to investigate the origin of this specific issue myself, but I did update a sample project to demonstrate it. 

[https://github.com/bigred8982/Swashbuckle.OData/tree/property_casing_sample](). 

The ```Customers``` model has been updated with sample "private" properties. They are still public properties in the C# model, but the EDM model was configured to ignore them. The config is specified in the ```ODataConfig.cs``` file.


Thanks for all the hard work!
